src/macaron/repo_finder/repo_finder_enums.py

```diff
@@ -0,0 +1,116 @@
+
+    #: Reported if the list of period separated tags that point to the SCM within the POM is not provided by the user in
+    # Macaron's config. E.g. scm.url, scm.connection
+    NO_POM_TAGS_PROVIDED = "No POM tags provided"
+
+    #: Reported if the user does not provide a version for the Repo Finder via the command line, or allow the version to
+    # be automatically discovered.
+    NO_VERSION_PROVIDED = "No version provided"
+
+    #: Reported if the user provides an unsupported type in the PURL command line argument.
+    UNSUPPORTED_PACKAGE_TYPE = "Unsupported package type"
+
+    #: Reported if there was an error reading the POM file.
+    POM_READ_ERROR = "POM read error"
+
+    #: Reported if the POM file contains no URLs within the SCM found at the provided tag locations.
+    SCM_NO_URLS = "SCM no URLs"
+
+    #: Reported if the POM file contains no VALID URLs within the SCM found at the provided tag locations. Validity is
+    # defined as any URL that resolves to a recognised version control system.
+    SCM_NO_VALID_URLS = "SCM no valid URLs"
+
+    #: Reported if the URL of the repository could not be reached.
+    HTTP_INVALID = "HTTP invalid"
+
+    #: Reported if the URL host returns a 404 (not found) for the sought resource.
+    HTTP_NOT_FOUND = "HTTP not found"
+
+    #: Reported if the URL host returns a 403 (forbidden) for the sought resource.
+    HTTP_FORBIDDEN = "HTTP forbidden"
+
+    #: Reported for all other bad status codes that a host could return. E.g. 500, etc.
+    HTTP_OTHER = "HTTP other"
+
+    #: Reported if deps.dev produces no response to the HTTP request.
+    DDEV_BAD_RESPONSE = "deps.dev bad response"
+
+    #: Reported if deps.dev returns JSON data that cannot be parsed.
+    DDEV_JSON_FETCH_ERROR = "deps.dev fetch error"
+
+    #: Reported if deps.dev returns JSON data that is missing expected fields.
+    DDEV_JSON_INVALID = "deps.dev JSON invalid"
+
+    #: Reported if deps.dev returns data that does not contain the desired SCM URL. E.g. The repository URL.
+    DDEV_NO_URLS = "deps.dev no URLs"
+
+    #: Reported if the provided PURL did not produce a result, but a more recent version could not be found.
+    NO_NEWER_VERSION = "No newer version than provided which failed"
+
+    #: Reported if the provided PURL did not produce a result, and the most recent version also failed.
+    LATEST_VERSION_INVALID = "A newer version than provided also failed"
+
+    #: Reported when a repository is found.
+    FOUND = "Found"
+
+    #: Reported when a repository is found via a parent POM.
+    FOUND_FROM_PARENT = "Found from parent"
+
+    #: Reported when a repository is found from a more recent version than was provided by the user.
+    FOUND_FROM_LATEST = "Found form latest"
+
+    #: Default value. Reported if the Repo Finder was not called. E.g. Because the repository URL was already present.
+    NOT_USED = "Not used"
+
+
+class CommitFinderOutcome(Enum):
```

Comment:
Same comment regarding the name of the class.

```suggestion
class CommitFinderMetadata(Enum):
```